src/backend/package.json

```diff
@@ -16,6 +16,7 @@
     "concat-stream": "^2.0.0",
     "cookie-parser": "^1.4.5",
     "cors": "^2.8.5",
+    "docker": "^1.0.0",
```

Comment:
you shouldn't have changes to any of these files, you should only be committing changes to the credits page. You can fix this by deleting the package-lock.json, comitting that change, and then deleting line 19 that you added in the package.json, running yarn install and then committing those changes